python/src/trezorlib/messages.py

```
@@ -8263,6 +8480,77 @@ def __init__(
         self.public_key = public_key
 
 
+class NeoGetAddress(protobuf.MessageType):
+    MESSAGE_WIRE_TYPE = 12301
+    FIELDS = {
+        1: protobuf.Field("address_n", "uint32", repeated=True, required=False, default=None),
+        2: protobuf.Field("show_display", "bool", repeated=False, required=False, default=None),
+    }
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)